at2-ns/src/user/full.rs

```diff
@@ -23,4 +25,9 @@ impl FullUser {
     pub fn keypair(&self) -> &sign::KeyPair {
         &self.keypair
     }
+
+    /// Return a thin version of the user
```

Comment:
Is this a good way to lose some pounds?

Can you describe in just one phrase what the difference between a `thin` and a `fat` user is?

Reply:
```
//! There is two types of user,
//! one having a KeyPair which is usually the current user,
//! and one having only a PublicKey, representing the other users on the network.
```

Both are "user", but if you find a better naming, go for it, I'm not really proud of it.

Reply:
`There are two types of users`

I think I called them `User` and `Contact`. The first has the private key, the second doesn't.

Reply:
Haa, well played, moving to `User` & `Contact`, thanks :)